### PR TITLE
Video Textures, Skybox, and Textured Platforms

### PR DESCRIPTION
**Pull request by alexottoboni**
@josepharhar I think I touched something that messed with the starting platform, probably platform scaling amounts. There were a couple merges and I might have overwritten something but I am having trouble finding it.

### CODE REVIEW

File: src/RhythmRunner.cpp [hunk 1]
```diff
@@ -15,11 +15,18 @@
...
 #include "json.hpp"
 #include "LevelJson.h"
 
 #define MUSIC "music/2.wav"
 
+std::shared_ptr<MenuState> menu_state;
+std::shared_ptr<GameState> game_state;
+
+GameUpdater game_updater;
+
```
Comment (josepharhar) [inline src/RhythmRunner.cpp]: Braden undid this to prevent an error on exit

File: src/RhythmRunner.cpp [hunk 2]
```diff
@@ -126,10 +133,10 @@ int main(int argc, char** argv) {
...
             level_generator = new LevelGenerator(menu_state->GetMusicPath());
           }
-          game_state = std::make_shared<GameState>(
-              level_generator->generateLevel(), std::make_shared<GameCamera>(),
-              std::make_shared<Player>());
-
+          game_state = std::make_shared<GameState>(level_generator->generateLevel(),
+                                           std::make_shared<GameCamera>(),
+                                           std::make_shared<Player>(),
+                                           std::make_shared<Sky>());
```
Comment (josepharhar) [inline src/RhythmRunner.cpp:134]: ?

File: src/RhythmRunner.cpp (hunk 1 above)
Comment (alexottoboni) [reply src/RhythmRunner.cpp]: Missed this when merging.

**Comment by bbeck13** [comment]
Fixed the starting platform thing and moved this to a new pr https://github.com/bbeck13/RhythmRunner/pull/114

**Pull request closed without merging**